devchat.go

```diff
@@ -89,6 +137,12 @@ var (
 		"1eab2de20e41abed903ab2f22e7ff56dc059666dbe2ebbce07a8afeece8d0424"} // Shok
 )
 
+func buildStyleApplicator(c color.Color) styleApplication {
```

Comment:
Make this take a pointer!

Reply:
Also might be a good idea to have a shorter name like mkStyle() but I can do that myself too